test/tests/functional/pbs_alps_release_tunables.py

```diff
@@ -96,12 +94,12 @@ def test_alps_release_wait_time(self):
         if len(out) >= 2:
             # variable 'out' is a list of tuples and every second element
             # in a tuple is the matched log message
-            time_prev = get_epoch(out[0][1])
+            time_prev = self.get_epoch(out[0][1])
             for data in out[1:]:
-                time_current = get_epoch(data[1])
+                time_current = self.get_epoch(data[1])
                 fail_msg = "alps_release_wait_time not working"
                 self.assertGreaterEqual(time_current - time_prev,
-                                        math.ceil(arwt),
+                                        math.floor(arwt),
```

Comment:
Now that we have high res timestamps, do we still need to make arwt into an integer?